app/models/aboutness.rb

```diff
@@ -1,5 +1,5 @@
 class Aboutness < ApplicationRecord
   belongs_to :work # work that is ABOUT something
   belongs_to :user
-  belongs_to :aboutable, polymorphic: true # something that the work is ABOUT
+  belongs_to :aboutable, polymorphic: true, optional: true # something that the work is ABOUT
```

Comment:
Here and below. Now by default all belong_to are mandatory, so we need to add `optional: true` where this relation is optional

Reply:
Het @abartov 
For most fields where I was not sure I've set field as `optional` only if we have records in database with this field being `null`.

In this particular case it is clearly valid, as we can have `Aboutness` referencing `wikidata_qid` instead of `Manifestation` or `Authority`.

Reply:
oh right! I forgot about that. :)